stratum/hal/config/x86-64-delta-ag9064v1-r0/chassis_config.pb.txt

```diff
@@ -9,714 +9,714 @@ nodes {
   index: 1
 }
 singleton_ports {
-  id: 48
+  id: 1
   name: "1/0"
   slot: 1
-  port: 48
+  port: 1
```

Comment:
@Yi-Tseng I think this port value is incorrect and does not work with the current Stratum logic. Can you take a look at this config?

Reply:
you are right, the `port` should be the front panel port number

Reply:
Thanks for confirming!